src/Tabs/README.md

```diff
@@ -120,3 +120,5 @@ notes: |
   <div>Hello I am the third panel</div>
 </Tabs.Deprecated>
 ```
+
+<guide />
```

Comment:
I think this supports `dataTestId`? Also, perhaps the `role="tablist"` might be worth including with `defaultText` here as well?